test-cases/features/hinted-handoff.yaml

```diff
@@ -11,7 +11,6 @@ instance_type_db: 'i3.2xlarge'
 # Options
 user_prefix: 'hinted-handoff'
 
-hinted_handoff_disabled: false
```

Comment:
hinted_handoff: 'enabled'

Reply:
But it's default value. Would you like to do it explicitly for this test?

Reply:
yes, since it is hinted handoff test yaml

Reply:
Fixed